### ISSUE TITLE
Add user/join leave sounds

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1926]
### Is your feature request related to a problem? Please describe.

The user join leave notification is really useful but I get distracted working on things or socializing so I may not see it it also looks really similar to the user online notification besides a small bit of text that says "user joined" at the bottom

### Describe the solution you'd like

I would like a sound for user joining and leaving they will be two different sounds so that it's easy to distinguish between joining and leaving when you're not directly paying attention

### Describe alternatives you've considered

Maybe creating a facet?

### Requesters

@epicEaston197

**Comment by shiftyscales** [comment]
This would require sound effects from @LacyDoes, and implementation of it by either @Frooxius or @ProbablePrime.

Seeking input from @Frooxius - do we have a known list of required sounds we could provide @LacyDoes with for notifications that do not yet have distinct sounds?

**Comment by epicEaston197** [comment]
I've rendered a short video demonstrating how the sound should depict joining and leaving
these two sound effects were used on multiple join leave interfaces created by the community don't know the sources of the audios
https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/76523692/9773859a-246e-46df-9445-f83974e76147

**Comment by troyBORG** [comment]
Can I add a request in here also to maybe have a string/URI field where you could put in a resdb link to an ogg file to you can overwrite it with your own custom sounds?

Like the: https://github.com/badhaloninja/UserJoinLeaveNotifications mod added?

**Comment by shadowpanther** [comment]
> Can I add a request in here also to maybe have a string/URI field where you could put in a resdb link to an ogg file to you can overwrite it with your own custom sounds?

That's #1928, which seems to be blocked by the notification system rework (#1918)

**Comment by shiftyscales** [comment]
@troyBORG - per our [reporting requirements](https://github.com/Yellow-Dog-Man/Resonite-Issues#reporting-requirements):
> - **Do not hijack issues for another feature/bug**, make another issue instead. Issues should never have comments of type "While you're at it, can you also do this?", it will not be considered and will get lost when the original problem in the issue gets solved

In this case, the linked issue shadowpanther included covers that issue.

**Comment by troyBORG** [comment]
> @troyBORG - per our [reporting requirements](https://github.com/Yellow-Dog-Man/Resonite-Issues#reporting-requirements):
> 
> > * **Do not hijack issues for another feature/bug**, make another issue instead. Issues should never have comments of type "While you're at it, can you also do this?", it will not be considered and will get lost when the original problem in the issue gets solved
> 
> In this case, the linked issue shadowpanther included covers that issue.

Sorry didn't think about that.   Also didn't know about the notification system rework. 😲

But I did notice in the update the user join/leave has toast only, audio only (which has none currently that I can hear), or both which only does the toast.

I was thinking if some users had already made join and leave noises and if new ones are created they may want to override it back to what they're used to and since the existing system I think is just hard linking to a resdb link for the sound file it plays, since we can edit stuff I thought it be nice to be able to see that url the settings is pointing to and just edit it if it's something we can see.  Even if it's something where you have to spawn out the settings thing that's in resonite essentials and edit it to see those variables that could be in there but not visible, as kind of a hidden advanced settings to not scare away new users with to many advanced options.

**Comment by epicEaston197** [comment]
Any updates on this issue?
I'm just wondering if the sounds have been created but just not implemented yet

**Comment by shiftyscales** [comment]
Lacy has confirmed these sounds have already been made, it'd just require engineering time to implement them. @Frooxius @ProbablePrime - would either of you have a moment to take this up? Otherwise, you could delegate to one of our other coders.

**Comment by Frooxius** [comment]
This is now out in 2024.8.5.1341!
